src/register.c

```diff
@@ -312,6 +312,20 @@ int io_uring_unregister_ring_fd(struct io_uring *ring)
 	return ret;
 }
 
+int io_uring_close_ring_fd(struct io_uring *ring)
+{
+	if (!(ring->features & IORING_FEAT_REG_REG_RING))
+		return -EOPNOTSUPP;
+	if (!(ring->int_flags & INT_FLAG_REG_RING))
+		return -EINVAL;
+	if (ring->ring_fd == -1)
+		return -EBADF;
+
+	close(ring->ring_fd);
+	ring->ring_fd = -1;
```

Comment:
Any code inside the `src/` directory should use the `__sys` function
wrapper to invoke a syscall. So this one should be
`__sys_close(ring->ring_fd);`.

Reply:
Fixed.